Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/DialogHelper.kt

```diff
@@ -92,6 +92,7 @@ object DialogHelper {
                     ) { _, _ ->
                         dialogInstance.negativeButtonFunction()
                     }
+                    setOnCancelListener { dialogInstance.negativeButtonFunction() }
```

Comment:
This affecting every dialog that uses `DialogHelper ` , right?
Can you make that change exclusively for the dialog in `SubmissionQRCodeScanFragment `?